### DISCUSSION TITLE
I can't register

### DISCUSSION THREAD

**Discussion by GuyFromNothing** [ppy/osu#23083] [Q&A] [answered]
### Type

Other

### Bug description

I tried to register and got an error.

### Screenshots or videos

![Screenshot_2023-04-02-10-16-53-121_sh ppy osulazer](https://user-images.githubusercontent.com/129651774/229340131-246d57fb-393a-41b6-a2d5-17bedfe163c2.jpg)

### Version

2021.1016.0-lazer

### Logs

.

**Comment by Joehuu** [comment, marked as answer]
Is there a reason why you're using a really old version? In any case, we don't support old versions.

**Reply by GuyFromNothing** [reply]
Oh, ok. I am downloading the new version. Thanks.
